android/build.gradle

```diff
@@ -45,7 +45,7 @@ def ExoplayerDependenciesList = [
 ]
 
 def ExoplayerDependencies = ExoplayerDependenciesList.collectEntries { property ->
-    [(property): safeExtGet(property)?.toBoolean() ?: false]
+    [(property): safeExtGet('RNVideo_' + property)?.toBoolean() ?: false]
```

Comment:
Isn't it breaking current behavior ? or is this fixing an issue ?

Reply:
You are right this can be breaking - I want to move all our properties to follow this `RNVideo_{NAME}` - will change it to support backward compatibility and update docs in other PR